sdks/python/apache_beam/io/avroio.py

```diff
@@ -451,7 +404,6 @@ class _FastAvroSource(filebasedsource.FileBasedSource):
   common to all file-based sources such as file-pattern expansion and splitting
   into bundles for parallel processing.
 
-  TODO: remove ``_AvroSource`` in favor of using ``_FastAvroSource``
   everywhere once it has been more widely tested
```

Comment:
```suggestion
```

has it been more widely tested? :)

Reply:
Perhaps @chamikaramj or @aaltay have an opinion on how mature the fast Avro IO is?

Reply:
I believe most of the users are still on the regular avro. If I remember correctly, for python3 the default is fastavro, for python2 the default is avro with a flag to change it.

@chamikaramj - what do you think?